tests/test_end_to_end.py

```diff
@@ -246,3 +261,57 @@ def test_get_export(self, client, export):
 
     def test_api_key(self, client):
         assert client.api_key()
+
+    def test_get_webhooks(self, client, webhook):
+        existent_webhooks = next(client.get_webhooks())
+
+        assert existent_webhooks['id']
+        assert existent_webhooks['status']
+        assert existent_webhooks['url']
+        assert existent_webhooks['events']
+
+    def test_create_webhook(self, client):
+        data = {
+            'url': 'https://fake.io/notes/closeio/',
+            'events': [{
+                'object_type': 'activity.note',
+                'action': 'created'
+            }]
+        }
+        new_webhook = client.create_webhook(data)
+
+        assert new_webhook['id']
+        assert new_webhook['status']
+        assert new_webhook['url']
+        assert new_webhook['events']
+
```

Comment:
I'd suggest to delete the webhook again here because this is a real end to end test that points to a closeio instance. If you just create the webhook, then it will remain there and the closeio instance will actually make requests to `fake.io` (which we should also change to `example.com`)